$OpenBSD$

Audio plugins not really expected to work, but do build.
It should be possible to get linux-v4l2 working with a bit more effort.
V+A are supported via ffmpeg.

Index: plugins/CMakeLists.txt
--- plugins/CMakeLists.txt.orig
+++ plugins/CMakeLists.txt
@@ -46,6 +46,8 @@ elseif("${CMAKE_SYSTEM_NAME}" MATCHES "FreeBSD")
 	add_subdirectory(linux-pulseaudio)
 	add_subdirectory(linux-v4l2)
 	add_subdirectory(linux-jack)
+elseif("${CMAKE_SYSTEM_NAME}" MATCHES "OpenBSD")
+	add_subdirectory(linux-capture)
 endif()
 
 if(WIN32 OR APPLE)
